packages/near-membrane-base/src/environment.ts

```diff
@@ -274,16 +274,18 @@ export class VirtualEnvironment {
         ownKeys: PropertyKeys,
         unforgeableGlobalThisKeys?: PropertyKeys
     ) {
-        const targetPointer = this.blueGetTransferableValue(target) as Pointer;
-        const args: Parameters<CallableInstallLazyPropertyDescriptors> = [targetPointer];
-        ReflectApply(ArrayProtoPush, args, ownKeys);
-        if (unforgeableGlobalThisKeys?.length) {
-            // Use `LOCKER_NEAR_MEMBRANE_UNDEFINED_VALUE_SYMBOL` to delimit
-            // `ownKeys` and `unforgeableGlobalThisKeys`.
-            args[args.length] = LOCKER_NEAR_MEMBRANE_UNDEFINED_VALUE_SYMBOL;
-            ReflectApply(ArrayProtoPush, args, unforgeableGlobalThisKeys);
+        if ((typeof target === 'object' && target !== null) || typeof target === 'function') {
```

Comment:
Just guarding `target` since it then going into cross-realm things.

Reply:
I'm not sure what this comment means, but I'm fine with the changes.